config.txt

```diff
@@ -1,11 +1,2 @@
-BASE_IMAGE_REPO=gcr.io/deeplearning-platform-release
-BASE_IMAGE_TAG=m122
-CPU_BASE_IMAGE_NAME=tf2-cpu.2-16.py310
-GPU_BASE_IMAGE_NAME=tf2-gpu.2-16.py310
-LIGHTGBM_VERSION=4.2.0
-TORCH_VERSION=2.4.0
-TORCHAUDIO_VERSION=2.4.0
-TORCHVISION_VERSION=0.19.0
-JAX_VERSION=0.4.26
 CUDA_MAJOR_VERSION=12
-CUDA_MINOR_VERSION=3
+CUDA_MINOR_VERSION=2
```

Comment:
I assume this downgrade is to align with Colab?

Reply:
Yes! Colab is on 12.2, so we are too now. Sucks to go backwards, but better that we're aligned.